Source/EmbeddedBoundary/WarpXInitEMField.cpp

```diff
@@ -0,0 +1,83 @@
+
+#include "WarpX.H"
+
+#ifdef AMREX_USE_EB
+#include <boost/math/special_functions/bessel.hpp>
+#endif
+
+amrex::RealArray
+WarpX::AnalyticSolSphere (amrex::Real x, amrex::Real y, amrex::Real z, amrex::Real t, amrex::Real r_sphere) {
+	amrex::Real k=2.7437/r_sphere;
+  	amrex::Real r = sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2));
+  	amrex::Real theta = atan2(sqrt(pow(x, 2) + pow(y, 2)), z);
+  	amrex::Real phi = atan2(y, x);
+  	amrex::Real H_r = 0;
+  	amrex::Real H_theta = 0;
+  	amrex::Real mu_r = PhysConst::mu0;
+#ifdef AMREX_USE_EB
+  	amrex::Real H_phi = k / (r_sphere * mu_r) * boost::math::sph_bessel(1, k*r) * sin(theta) * cos(k * PhysConst::c * t);
```

Comment:
Discussed on Slack:

yes we do not depend on boost because it's a pretty heavy dependency to pull in. But if you only need them on the host-side, you can take the C++17 functions!
https://en.cppreference.com/w/cpp/numeric/special_functions/sph_bessel

We are compiling with C++14 or newer, but we can with no problem require C++17 for selected features already! (We will probably transition to require C++17 anyway during the year.)

So it would be no problem to say e.g. "if you want to use embedded boundaries, use we need a more recent compiler", absolutely no problem.